cli.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import datetime
+
+from flow_client import settings, sig_gen
+
+
+def main():
+
+    description = """Generate a url with signed request object for a flow endpoint.
+
+    flow can be selected by name from list in config or specified by url. if not specified, defaults to first flow in config.
+    time is specified in days and seconds. defaults to time set in config which is usually 1 day"""
+
+    parser = argparse.ArgumentParser(
+        description=description,
+        prog="sign_url.sh",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    parser.add_argument(
+        "-n",
+        "--name",
+        help="flow name. used to select flow from list of flows in config",
+    )
+    parser.add_argument("-u", "--url", help="flow url")
+    parser.add_argument(
+        "-d", "--days", type=float, help="num days sig is valid. summed with seconds"
+    )
+    parser.add_argument(
+        "-s", "--seconds", type=float, help="num seconds sig is valid. summed with days"
+    )
+
+    args = parser.parse_args()
+
+    if args.seconds is None and args.days is None:
+        valid_for = settings.VALID_FOR
+        print(f"using default duration:")
+    else:
+        valid_for = datetime.timedelta(
+            days=0 if args.days is None else args.days,
+            seconds=0 if args.seconds is None else args.seconds,
+        )
```

Comment:
This seems like a lot. Is it reasonable to set the default value of `seconds` and `days` to `settings.VALID_FOR.seconds`? That way we avoid defaulting to zero with an in-line if statement and most of this if/else logic goes away.
```suggestion
    parser.add_argument(
        "-d", "--days", type=float, help="num days sig is valid. summed with seconds", default=settings.VALID_FOR.days
    )
    parser.add_argument(
        "-s", "--seconds", type=float, help="num seconds sig is valid. summed with days", default=settings.VALID_FOR.seconds
    )

    args = parser.parse_args()

    valid_for = datetime.timedelta(
        days=args.days,
        seconds=args.seconds,
    )
```